requirements.txt

```diff
@@ -0,0 +1,27 @@
+questionary==2.0.1
```

Comment:
Can you remove this diff (.txt file) from the PR? We're using poetry for package management so if you want to add new packages, you need to run `poetry add` to do the dependency resolution.

We can also do this for you - you just need to tell us what extra packages this PR needs (which I don't think is any actually).

Reply:
Sure I'll remove it asap